packages/jellyfish-api-core/__tests__/category/loan/paybackLoan.test.ts

```diff
@@ -753,13 +819,14 @@ describe('paybackLoan failed', () => {
     await expect(promise).rejects.toThrow('tx must have at least one input from token owner')
   })
 
-  it('should fail paybackLoan if resulted in zero interest loan', async () => {
-    const promise = bob.rpc.loan.paybackLoan({
-      vaultId: bobVaultId,
-      amounts: '40@TSLA',
-      from: bobloanAddr
-    })
-    await expect(promise).rejects.toThrow(RpcApiError)
-    await expect(promise).rejects.toThrow('Cannot payback this amount of loan for TSLA, either payback full amount or less than this amount!')
-  })
+  // this test on longer applies after FortCanningHill
```

Comment:
you can remove this one, But make sure to add a test so that the loan can continue with a sub satoshi `interestPerBlock` values after a payback. This might help with https://github.com/DeFiCh/jellyfish/issues/961#issuecomment-1012715731

Ref - https://github.com/DeFiCh/ain/blob/c31d2b3997f294a1d1d63db9fffed5b6e6d6ab13/src/masternodes/loan.cpp#L348

Reply:
reminder to self